client/src/pages/TakeSurvey/SurveyClosedMessage/SurveyClosedMessage.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+
+import { Typography } from '@material-ui/core'
+
+const SurveyClosedMessage = () => {
+  return <Typography variant='h3'>This survey is closed and is no longer accepting responses.</Typography>
```

Comment:
Could just be an arrow function instead of using a return!

Reply:
done